core/src/test/java/io/camunda/migrator/CamundaClientConfigurationTest.java

```diff
@@ -21,7 +21,7 @@
     "camunda.client.grpc-address=http://helloworld:33333",
     "camunda.client.rest-address=http://helloworld:44444"
 })
-public class CamundaClientConfigurationIT {
+public class CamundaClientConfigurationTest {
```

Comment:
Looking at the logs, I still see this test not being executed? 🤔 

I think the reason for that is we need to add the surefire plugin to the plugins in the pom.xml of the core module:

```xml
      <plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-surefire-plugin</artifactId>
        <version>3.5.3</version>
      </plugin>
```

Can you take care?

Reply:
Added ✔️  Lets see